yb-voyager/cmd/import.go

```diff
@@ -537,8 +537,8 @@ func validateOnPrimaryKeyConflictFlag() error {
 	}
 
 	// restrict setting --enable-upsert as true if --on-primary-key-conflict is not set to ERROR
-	if tconf.EnableUpsert && tconf.OnPrimaryKeyConflictAction != constants.PRIMARY_KEY_CONFLICT_ACTION_ERROR {
-		return fmt.Errorf("--enable-upsert=true can only be used with --on-primary-key-conflict=ERROR")
+	if tconf.EnableUpsert && tconf.OnPrimaryKeyConflictAction != constants.PRIMARY_KEY_CONFLICT_ACTION_ERROR_POLICY {
+		return fmt.Errorf("--enable-upsert=true can only be used with --on-primary-key-conflict=ERROR-POLICY")
 	}
```

Comment:
can you help me remember the context here?
and also think with case of stash and continue

Reply:
--enable-upsert true and on-primary-key-conflict ignore would be conflicting, therefore we only allow it if on-primary-key-conflict is set to ERROR-POLICY. added comment in code. 

enable-upsert will eventually be deprecated, so not really thinking much about stash-and-continue and enable-upsert. But either way, enable-upsert=true + error-policy=stash-and-continue just means that there will not be any conflict errors in the first place, so error-policy will kick in for all the rest of the errors. Don't see any confusion here. Lmk if you see any potential confusion with these flags.